<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackInfosView</class>
 <widget class="QWidget" name="TrackInfosView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="15" column="2" colspan="3">
    <widget class="QLineEdit" name="d_releaseDate"/>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>BPM</string>
     </property>
    </widget>
   </item>
   <item row="18" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <widget class="QLineEdit" name="d_bpm"/>
   </item>
   <item row="18" column="4">
    <widget class="QLineEdit" name="d_key"/>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Release Date</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QLineEdit" name="d_name"/>
   </item>
   <item row="10" column="2" colspan="3">
    <widget class="QLineEdit" name="d_publishdate"/>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="QLineEdit" name="d_mixname"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Publish Date</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Mix Name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="16" column="2" colspan="3">
    <widget class="QLineEdit" name="d_label"/>
   </item>
   <item row="17" column="4">
    <widget class="QLineEdit" name="d_length"/>
   </item>
   <item row="17" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QLineEdit" name="d_price"/>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Release</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="3">
    <widget class="QLineEdit" name="d_release"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Artist(s)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Remixer(s)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Full Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="d_artists"/>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="d_remixers"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="d_title"/>
   </item>
   <item row="1" column="4" rowspan="3">
    <widget class="QLabel" name="imageArea">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
